x-pack/plugins/graph/public/helpers/saved_workspace_utils.ts

```diff
@@ -123,11 +123,11 @@ export async function getSavedWorkspace(
   return savedObject as GraphWorkspaceSavedObject;
 }
 
-export function deleteSavedWorkspace(
+export async function deleteSavedWorkspace(
   savedObjectsClient: SavedObjectsClientContract,
   ids: string[]
 ) {
-  return Promise.all(ids.map((id: string) => savedObjectsClient.delete(savedWorkspaceType, id)));
+  await Promise.all(ids.map((id: string) => savedObjectsClient.delete(savedWorkspaceType, id)));
```

Comment:
nitpick: why change this code here?

As of now we do not use the promise reference, but it would be useful to have it in case we want to in the future - and the code already allowed it.

Reply:
You are right, I have returned this.